Assets/MRTK/SDK/Inspectors/UX/Interactable/ButtonConfigHelperInspector.cs

```diff
@@ -69,6 +78,25 @@ public override void OnInspectorGUI()
             bool iconFoldout = SessionState.GetBool(IconFoldoutKey, true);
             bool showComponents = SessionState.GetBool(ShowComponentsKey, false);
 
+            if (cb.EditorCheckForCustomIcon())
+            {
+                using (new EditorGUILayout.VerticalScope(EditorStyles.helpBox))
+                {
+                    EditorGUILayout.LabelField("Custom Icon Migration", EditorStyles.boldLabel);
+                    EditorGUILayout.HelpBox(customIconUpgradeMessage, MessageType.Error);
+
+                    using (new EditorGUILayout.HorizontalScope())
+                    {
+                        if (GUILayout.Button("Use migration tool to upgrade buttons"))
+                        {
+                            EditorApplication.ExecuteMenuItem("Mixed Reality Toolkit/Utilities/Migration Window");
```

Comment:
Ah, didn't know you could do this. Cool!

Reply:
It works well enough in this case but I _really_ wish I could specify which migration handler type to use when opening the helper. Unfortunately this inspector and the migration helper can't take dependencies on one another.